test/e2e/bootstrap/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
```

Comment:
Perfect, then add @luxas @jbeda and yourself initially, and we'll see if there are other volunteers